dsr_gazebo2/launch/dsr_gazebo.launch.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 from launch import LaunchDescription
-from launch.actions import RegisterEventHandler,DeclareLaunchArgument, IncludeLaunchDescription
+from launch.actions import RegisterEventHandler,DeclareLaunchArgument, IncludeLaunchDescription, TimeAction

```

Comment:
I found that ros2 does not has TimeAction. You should use TimerAction. 

I think its a typo from you end.

Reply:
Hi @gautham-nf ,
Thank you so mush :)